framework/behaviors/BlameableBehavior.php

```diff
@@ -96,9 +96,8 @@ public function init()
      */
     protected function getValue($event)
     {
-        if ($this->value === null) {
-            $user = Yii::$app->get('user', false);
-            return $user && !$user->isGuest ? $user->id : null;
+        if ($this->value === null && Yii::$app->has('user')) {
```

Comment:
`!$user->isGuest` check is missing. Because of that behavior is not the same as it was.